flying-saucer-core/src/main/java/org/xhtmlrenderer/css/parser/property/SizePropertyBuilder.java

```diff
@@ -94,7 +94,7 @@ public List<PropertyDeclaration> buildDeclarations(
             } else {
                 throw new CSSParseException("Value for " + cssName + " must be a length or identifier", -1);
             }
-        } else { /* values.size == 2 */
+        } else if (values.size() == 2) { /* values.size == 2 */
```

Comment:
Is this comment is still needed?
It just duplicates the code.

Reply:
cleaned, sorry (modification made in plaintext editor)